### ISSUE TITLE
Should we recommend 3 days, or 7 days?

### ISSUE THREAD

**Issue by ericmeltzer** [ericmeltzer/disaster-survival-manual#1]
Initially I thought 3 days/72 hours was the amount of time we'd be trying to support people for with our manual and equipment. Steve Taffee thinks it should be 7, and made some convincing arguments to that effect. Anyone else have an opinion about this?

**Comment by peterh-capella** [comment]
The word on ready.gov is the 3 days/72 hours you indicate, but they also say "You could get help in hours or it might take days." My initial thought on 7 days is that the additional food needed might get too heavy. (The gear would largely remain the same or increment slightly for things like water tabs, toothpaste, hygiene products.) So weighing the difference and checking the volume of additional food would be a deciding factor.

**Comment by ericmeltzer** [comment]
@peterh-capella Yeah that makes sense--the main things that scale are food and water. I say we stick with an overall timeframe of 72 hours, but that we tell people to store more food and water at home (outside of their kit) since its easy to do and doesn't affect portability of the kit itself
